File: ios/PouleParty/Models/MyGame.swift
//
//  MyGame.swift
//  PouleParty
//
//  A game shown in the "My Games" list, tagged with the user's role (creator or hunter).
//

import Foundation

struct MyGame: Equatable, Identifiable {
    let game: Game
    let role: GameRole

    var id: String { game.id }
}

File: android/app/src/main/java/dev/rahier/pouleparty/model/MyGame.kt
package dev.rahier.pouleparty.model

/**
 * A game shown in the "My Games" list, tagged with the user's role (creator or hunter).
 */
enum class MyGameRole { CREATOR, HUNTER }

data class MyGame(
    val game: Game,
    val role: MyGameRole
)
